docs-client/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@material-ui/core": "^4.10.2",
     "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.56",
```

Comment:
They use a different scheme for `lab`.
https://material-ui.com/components/about-the-lab/